### PR TITLE
Update Dockerfile build

### PR DESCRIPTION
**Pull request by chancez**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -73,10 +73,8 @@ COPY presto-password-authenticators /build/presto-password-authenticators
 COPY src /build/src
 COPY pom.xml /build/pom.xml
 
-# Install presto-server
-RUN cd /build/presto-server && mvn -B -e -T 1C -DskipTests -DfailIfNoTests=false -Dtest=false package
-# Install presto-cli
-RUN cd /build/presto-cli && mvn -B -e -T 1C -DskipTests -DfailIfNoTests=false -Dtest=false package
+# build presto
```
Comment (bentito) [inline Dockerfile]: So in general: We're not doing it like the Presto dockerization any more, just copying the binaries, we're always going to build it to include source changes?

The command line change to mvn <- lgtm

Comment (chancez) [reply Dockerfile]: That's always been the case, we've built it from source for quite a while, but I suppose how we've been doing it does potentially pull in some stuff that's prebuilt yes.

Comment (bentito) [reply Dockerfile]: /lgtm

**Pull request merged**